attributes/default.rb

```diff
@@ -20,5 +20,6 @@
 
 default['hashicorp-vault']['config']['path'] = '/etc/vault/vault.json'
 default['hashicorp-vault']['config']['address'] = '127.0.0.1:8200'
+default['hashicorp-vault']['config']['log_level'] = 'info'
```

Comment:
Making this default for two reasons:

1. Explicit is better than implicit
2. Easier on the fly modification of log level via config